docs/google.rst

```diff
@@ -302,6 +296,36 @@ Example::
 .. Note:: A container image must be specified to deploy the process execution. You can use a different Docker image for
   each process using one or more :ref:`config-process-selectors`. 
 
+Process definition
+------------------
+Processes can be defined as usual and by default the ``cpus`` and ``memory`` directives are used to instantiate a custom
+machine type with the specified compute resources.  If ``memory`` is not specified, 1GB of memory is allocated per cpu.
+
+The process ``machineType`` directive may optionally be used to specify a predifined Google Compute Platform `machine type <https://cloud.google.com/compute/docs/machine-types>`_
+If specified, this value overrides the ``cpus`` and ``memory`` directives.
+If the ``cpus`` and ``memory`` directives are used, the values must comply with the allowed custom machine type `specifications <https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type#specifications>`_ .  Extended memory is not directly supported, however high memory or cpu predefined
+instances may be utilized using the ``machineType`` directive
+
+Examples::
+
+    process custom_instance_using_cpus_and_memory {
+        cpus 8
+        memory '40 GB'
+
+        """
+        <Your script here>
+        """
+    }
+
+    process predefined_instance_overrides_cpus_and_memory {
+        cpus 8
+        memory '40 GB'
+        machineType 'n1-highmem-8'
```

Comment:
I think it would be clear just putting `machineType` here, to stress that's an alternative setting. I can do if you agree.

Reply:
Sounds good, thanks!